plot-api/src/commonMain/kotlin/jetbrains/letsPlot/LetsPlot.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot
+
+import jetbrains.letsPlot.frontend.NotebookFrontendContext
+import jetbrains.letsPlot.intern.OptionsMap
+
+object LetsPlot {
+    var frontendContext: FrontendContext = createDefaultFrontendContext()
+
+    @Suppress("MemberVisibilityCanBePrivate")
+    var apiVersion: String = "Unknown"
+
+    fun setTheme(theme: OptionsMap?) {
```

Comment:
Lets's make it a property - similar to `frontendContext`.

Reply:
Done